aws/table_aws_kinesis_firehose_delivery_stream.go

```diff
@@ -0,0 +1,277 @@
+			},
+			{
+				Name:        "delivery_stream_type",
+				Description: "The delivery stream type.",
+				Type:        pb.ColumnType_STRING,
+				Hydrate:     describeStream,
+				Transform:   transform.FromField("DeliveryStreamDescription.DeliveryStreamType"),
+			},
+			{
+				Name:        "version_id",
+				Description: "The version id of the stream. Each time the destination is updated for a delivery stream, the version ID is changed, and the current version ID is required when updating the destination",
+				Type:        pb.ColumnType_STRING,
+				Hydrate:     describeStream,
+				Transform:   transform.FromField("DeliveryStreamDescription.VersionId"),
+			},
+			{
+				Name:        "create_timestamp",
+				Description: "The date and time that the delivery stream was created.",
+				Type:        pb.ColumnType_TIMESTAMP,
+				Hydrate:     describeStream,
+				Transform:   transform.FromField("DeliveryStreamDescription.CreateTimestamp"),
+			},
+			{
+				Name:        "last_update_timestamp",
+				Description: "The date and time that the delivery stream was last updated.",
+				Type:        pb.ColumnType_TIMESTAMP,
+				Hydrate:     describeStream,
+				Transform:   transform.FromField("DeliveryStreamDescription.LastUpdateTimestamp"),
+			},
+			{
+				Name:        "has_more_destinations",
+				Description: "Indicates whether there are more destinations available to list.",
+				Type:        pb.ColumnType_BOOL,
+				Hydrate:     describeStream,
+				Transform:   transform.FromField("DeliveryStreamDescription.HasMoreDestinations"),
+			},
+			{
+				Name:        "delivery_stream_encryption_configuration",
+				Description: "Indicates the server-side encryption (SSE) status for the delivery stream.",
+				Type:        pb.ColumnType_JSON,
+				Hydrate:     describeStream,
+				Transform:   transform.FromField("DeliveryStreamDescription.DeliveryStreamEncryptionConfiguration"),
+			},
+			{
+				Name:        "source",
+				Description: "If the DeliveryStreamType parameter is KinesisStreamAsSource, a SourceDescription object describing the source Kinesis data stream.",
+				Type:        pb.ColumnType_JSON,
+				Hydrate:     describeStream,
+				Transform:   transform.FromField("DeliveryStreamDescription.Source"),
+			},
+			{
+				Name:        "destinations",
+				Description: "The destinations for the stream.",
+				Type:        pb.ColumnType_JSON,
+				Hydrate:     describeStream,
+				Transform:   transform.FromField("DeliveryStreamDescription.Destinations"),
+			},
+			{
+				Name:        "failure_description",
+				Description: "Provides details in case one of the following operations fails due to an error related to KMS: CreateDeliveryStream, DeleteDeliveryStream, StartDeliveryStreamEncryption,StopDeliveryStreamEncryption.",
+				Type:        pb.ColumnType_JSON,
+				Hydrate:     describeStream,
+				Transform:   transform.FromField("DeliveryStreamDescription.FailureDescription"),
+			},
```

Comment:
leaving the first 5 columns, can you please arrange the remaining columns in alphabetical order